src/components/select/select.scss

```diff
@@ -11,6 +11,12 @@
     display: none;
 }
 
+:host {
+    display: block;
+    position: relative;
+    height: $height-of-mdc-text-field;
+}
```

Comment:
```suggestion
}

:host([hidden]) {
    display: none;
}
```

Reply:
Ah, this one already had this the `:host([hidden])`, but it comes before this new code. It shouldn't matter for the rendering, but consistency is nice when reading the code, so I'll swap the order around 🙂

Reply:
⚡